modules/ROOT/pages/create-custom-rulesets.adoc

```diff
@@ -4,7 +4,7 @@ include::partial$api-gov-ruleset-discovery.adoc[]
 
 If you need a ruleset other than what is provided, you can submit your ideas for rulesets you would like MuleSoft to provide in future releases on the https://help.mulesoft.com/s/ideas[MuleSoft Ideas Portal^]. 
 
-Rulesets are implemented using AMF, an open-source framework for managing metadata documents. AMF is used to parse, transform, validate and render arbitrary metadata models, with built-in features for API models. AMF is used by Anypoint Platform to parse RAML, OAS, AsyncAPI and GraphQL API specifications.
+Rulesets are implemented using AMF, an open-source framework for managing metadata documents. AMF is used to parse, transform, validate, and render arbitrary metadata models, with built-in features for API models. AMF is used by Anypoint Platform to parse RAML, OAS, AsyncAPI, and GraphQL API specifications.
```

Comment:
Suggestion to remove one occurrence of passive voice: 
Anypoint Platform uses AMF to...